### PR TITLE
Expose dockerfile and args arguments under build 

### PR DESCRIPTION
**Pull request by jashparekh**

### CODE REVIEW

File: python_on_whales/components/compose/models.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ class DependencyCondition(BaseModel):
 
 class ComposeServiceBuild(BaseModel):
     context: Optional[Path] = None
+    dockerfile: Optional[str] = None
+    args: Optional[dict] = None
```
Comment (gabrieldemarmiesse) [inline python_on_whales/components/compose/models.py:41]:
```suggestion
    dockerfile: Optional[Path] = None
    args: Optional[dict[str, Any]] = None
```

**Review by gabrieldemarmiesse — CHANGES_REQUESTED** [review]
Thanks for the pull request!!! Can we just use more precise types? Thanks!

**Comment by jashparekh** [comment]
> Thanks for the pull request!!! Can we just use more precise types? Thanks!

Made the requested changes, @gabrieldemarmiesse can you take another look?

**Comment by jashparekh** [comment]
Hey @gabrieldemarmiesse - Following up for your review on this PR

**Comment by gabrieldemarmiesse** [comment]
Sure, I'll take a look!

**Review by gabrieldemarmiesse — APPROVED** [review]
Very cool :)

**Comment by jashparekh** [comment]
Thank you @gabrieldemarmiesse, can you also cut a new release with this change?

**Comment by gabrieldemarmiesse** [comment]
We'll first merge your other PR about pyproject.toml :)

**Pull request merged**